e2e-tests/signing.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eEuxo pipefail
+
+unset CARGO_INSTALL_ROOT
+
+CARGO_HOME=$(mktemp -d 2>/dev/null || mktemp -d -t 'cargo-home')
+export CARGO_HOME
+export PATH="$CARGO_HOME/bin:$PATH"
+
+python server.py 2>/dev/null &
```

Comment:
Ks it possible to avoid having a separate python file at all:

```suggestion
python -m http.server 4443 2>/dev/null &
```

Reply:
Not for https, which is what I want here.